presto-iceberg/src/test/java/com/facebook/presto/iceberg/TestIcebergMetadataListing.java

```diff
@@ -153,4 +153,16 @@ public void testTableValidation()
         assertQuerySucceeds("SELECT * FROM iceberg.test_schema.iceberg_table1");
         assertQueryFails("SELECT * FROM iceberg.test_schema.hive_table", "Not an Iceberg table: test_schema.hive_table");
     }
+
+    @Test
+    public void testRenameView()
+    {
+        assertQuerySucceeds("CREATE TABLE iceberg.test_schema.iceberg_test_table (_string VARCHAR, _integer INTEGER)");
+        assertUpdate("CREATE VIEW iceberg.test_schema.test_view_to_be_renamed AS SELECT * FROM iceberg.test_schema.iceberg_test_table");
+        assertUpdate("ALTER VIEW IF EXISTS iceberg.test_schema.test_view_to_be_renamed RENAME TO iceberg.test_schema.test_view_renamed");
```

Comment:
Can you please add 2 more test here when table doesn't exist. One with `IF EXISTS` and one without it?